db/migrate/20201203193021_relate_rentalsto_custand_videos.rb

```diff
@@ -0,0 +1,6 @@
+class RelateRentalstoCustandVideos < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :rentals, :customer, foreign_key: true
+    add_reference :rentals, :video, foreign_key: true
+  end
+end
```

Comment:
if we already have columns for video_id and customer_id, I don't think we need the foreign keys